BAEL-2360

MajewskiKrzysztof opened the pull request.

jzheaux commented on spring-boot-mvc/src/main/java/com/baeldung/shedlock/ShedlockScheduler:
```diff
@@ -0,0 +1,15 @@
+package com.baeldung.shedlock;
```
Why isn't this a Java file?

jzheaux replied:
Also, can we make sure it matches the article?

jzheaux commented on spring-boot-mvc/pom.xml:
```diff
@@ -72,6 +72,18 @@
             <version>${spring.fox.version}</version>
         </dependency>
 
+        <!-- ShedLock -->
```
It feels weird that this is in `spring-boot-mvc`. Is there a more general one? My concern is this has nothing to do with MVC.

MajewskiKrzysztof commented:
@jzheaux Sorry for that mistake :/
I made changes. I also deleted Application class and created new one: SchedulerConfiguration. I moved all to spring-all project. 
Is it ok now?
